packages/access-api/src/utils/context.js

```diff
@@ -36,6 +36,14 @@ export function getContext(request, env, ctx) {
           sender: config.POSTMARK_SENDER,
         })
 
+  if (['staging', 'production'].includes(env.ENV)) {
```

Comment:
```suggestion
  if (['staging', 'production'].includes(config.ENV)) {
```

looks like this is used that way before?

this config with `context.js` and `config.js` is super confusing. We should later on just have one file